pallets/parachain-staking/src/lib.rs

```diff
@@ -642,18 +756,23 @@ pub mod pallet {
 		// Nominator Does Not Exist
 		NominatorDNE,
 		CandidateDNE,
+		NominationDNE,
 		NominatorExists,
 		CandidateExists,
 		ValBondBelowMin,
 		NomBondBelowMin,
 		NominationBelowMin,
 		AlreadyOffline,
 		AlreadyActive,
-		AlreadyLeaving,
-		CannotActivateIfLeaving,
+		CandidateAlreadyScheduledToLeave,
+		NominatorAlreadyLeaving,
+		NominationAlreadyRevoked,
+		CandidateAlreadyLeaving,
```

Comment:
what is the difference between `CandidateAlreadyScheduledToLeave` and `CandidateAlreadyLeaving`?

Reply:
Fixing now, good catch